qiskit/quantum_info/operators/pauli.py

```diff
@@ -256,10 +258,19 @@ def sgn_prod(p1, p2):
         return new_pauli, phase
 
     @property
-    def numberofqubits(self):
+    def num_qubits(self):
         """Number of qubits."""
         return len(self)
 
+    @property
+    def numberofqubits(self):
+        """Deprecated, use ``num_qubits`` instead. Number of qubits."""
+        warnings.warn('The Pauli.numberofqubits method is deprecated as of 0.14.0, and '
```

Comment:
We're still targetting this for 0.13.0:

```suggestion
        warnings.warn('The Pauli.numberofqubits method is deprecated as of 0.13.0, and '
```

Reply:
Thanks for noticing 👍